langchain/scripts/create-entrypoints.js

```diff
@@ -129,6 +129,7 @@ const entrypoints = {
   "stores/file/node": "stores/file/node",
   "stores/message/dynamodb": "stores/message/dynamodb",
   "stores/message/redis": "stores/message/redis",
+  "stores/message/redis_upstash": "stores/message/redis_upstash",
```

Comment:
Can we reverse the order of all these names to `upstash_redis` to mirror the company's name?